Mu2eG4/geom/crv_counters_extracted_v01.txt

```diff
@@ -16,8 +16,8 @@ double          crs.scintillatorBarLengthT2      =  3200; //2   CRV-L endcap mod
 double          crs.scintillatorBarThickness     =  19.8; //mm
 double          crs.scintillatorBarWidth         =  51.3; //mm
 double          crs.layerOffset                  =  42;   //mm
-double          crs.gapLarge                     =   0.5; //mm
-double          crs.gapSmall                     =   0.0875; //mm
+double          crs.gapLarge                     =   0.2;   //mm  //0.366 according to crv_parameters.xlsx
```

Comment:
What's going on here - the value of the parameter is different than the value given in the reference?
This confuses me.

Reply:
Our Wideband studies found that a gap of 0.2mm matches our data better than the previously estimated 0.366mm. That's why we want to go forward with 0.2mm for the KPP simulations. We are continuing our Wideband studies, but we think that a 0.2mm effective gap is the best value to use. The main source of the disagreement is the non-uniform thickness of the TiO2 coating of the CRV counters. We will eventually make new CRV lookup tables that will take our findings into consideration.

Reply:
Is a change actually requested here? Ralf answered your question. Should we block this PR because of a comment?